components/pipeline-service/staging/stone-stg-m01/deploy.yaml

```diff
@@ -1205,7 +1209,7 @@ spec:
       containers:
       - args:
         - -zap-log-level=6
-        image: quay.io/redhat-appstudio/pipeline-service-exporter:2782659983c94692497467cd1cf952b1bc1f0da4
+        image: quay.io/redhat-appstudio/pipeline-service-exporter:d0109787b251d347f08f5d5e473af7ef5dec54b2
```

Comment:
sha is up at https://quay.io/repository/redhat-appstudio/pipeline-service-exporter?tab=tags